### PR TITLE
Fix punctuation splitting

### LINKED ISSUE #95: No ending punctuation issue
If there is not an ending punctuation, there is no temp output chunk, which leads to this error:

image

I don't think this is the case for all inputs, so here is the input to demo this issue:

Input text that fails
```Avsikten hos gemenskapen är att utgå från ett ‘globalt perspektiv’, där förhållanden i Sverige inte ska ses som normerande eller viktigare än de i omvärlden```

Input text that works
```Avsikten hos gemenskapen är att utgå från ett ‘globalt perspektiv’, där förhållanden i Sverige inte ska ses som normerande eller viktigare än de i omvärlden.```

### PR DESCRIPTION
**Pull request by andrew-fennell**
Fixes #95 - No punctuation ```split_text()``` error.

Closes #95

### CODE REVIEW

**Review by JacobSmith1020 — APPROVED** [review]
Working correctly, looks good :)

**Comment by andrew-fennell** [comment]
> Working correctly, looks good :)

Awesome! I'll wait for CI to pass before merging.

**Pull request merged**
